package.json

```
@@ -62,8 +64,9 @@
     "pathpida": "^0.20.1",
     "prettier": "^2.8.3",
     "sharp": "^0.32.1",
-    "storybook": "^7.0.26",
+    "ts-node": "^10.9.1",
     "typescript": "4.9.5",
+    "storybook": "^7.0.26",
```

名前順だとStorybookがts-nodeよりも上に来るきがする🤔
```suggestion
    "storybook": "^7.0.26",
    "ts-node": "^10.9.1",
    "typescript": "4.9.5",
```

---

そういう順なのかこれｗ

---

https://github.com/uyupun/official/pull/246/commits/69c55bd6537d204e9edff1cc78a96a94d0ca06c9 で対応しました

---

あざます！
特に自分でpackage.jsonに記述せずに、コマンドからインストールする場合は基本名前順としてpackage.jsonに記述されていくんですよ（多分今回コンフリクト起こしてて、明記したからかな〜という予想）